src/osversion/VersionTranslator.cpp

```diff
@@ -7608,6 +7613,267 @@ namespace osversion {
+          info.maxCyclingRate = *val_;
+        }
+
+        // 39 - Heat Pump Time Constant
+        if (auto val_ = object.getDouble(39)) {
+          info.heatPumpTimeConst = *val_;
+        }
+
+        // 40 - Fraction of On-Cycle Power Use
+        if (auto val_ = object.getDouble(40)) {
+          info.fractionOnCycle = *val_;
+        }
+
+        // 41 - Heat Pump Fan Delay Time
+        if (auto val_ = object.getDouble(41)) {
+          info.hpDelayTime = *val_;
+        }
+
+      } else if (iddname == "OS:ZoneHVAC:WaterToAirHeatPump") {
+
+        std::string heatingCoilType;
+        std::string heatingCoilName;
+        std::string coolingCoilType;
+        std::string coolingCoilName;
+
+        // 13 - Heating Coil Name
+        if (auto handleStr_ = object.getString(13)) {
+          if (auto coilObject_ = idf_3_6_1.getObject(toUUID(*handleStr_))) {
+            heatingCoilName = coilObject_->nameString();
+            heatingCoilType = coilObject_->iddObject().name();
+          }
+        }
+
+        // 14 - Cooling Coil Name
+        if (auto handleStr_ = object.getString(14)) {
+          if (auto coilObject_ = idf_3_6_1.getObject(toUUID(*handleStr_))) {
+            coolingCoilName = coilObject_->nameString();
+            coolingCoilType = coilObject_->iddObject().name();
+          }
+        }
+
+        const bool hasHeatingChanges =
+          (std::find(heatingCoilTypesChanged.cbegin(), heatingCoilTypesChanged.cend(), heatingCoilType) != heatingCoilTypesChanged.end());
+        const bool hasCoolingChanges =
+          (std::find(coolingCoilTypesChanged.cbegin(), coolingCoilTypesChanged.cend(), coolingCoilType) != coolingCoilTypesChanged.end());
+        const bool infoNeeded = hasHeatingChanges || hasCoolingChanges;
+
+        if (!infoNeeded) {
+          continue;
+        }
+
+        CoilLatentTransitionInfo& info = result.emplace_back();
+        info.parentType = CoilLatentTransitionInfo::ParentType::ZoneWAHP;
+        info.parentName = object.nameString();
+        info.heatingCoilName = std::move(heatingCoilName);
+        info.heatingCoilType = std::move(heatingCoilType);
+        info.coolingCoilName = std::move(coolingCoilName);
+        info.coolingCoilType = std::move(coolingCoilType);
+
+        // 15 - Maximum Cycling Rate
+        if (auto val_ = object.getDouble(15)) {
+          info.maxCyclingRate = *val_;
+        }
+
+        // 16 - Heat Pump Time Constant
+        if (auto val_ = object.getDouble(16)) {
+          info.heatPumpTimeConst = *val_;
+        }
+
+        // 17 - Fraction of On-Cycle Power Use
+        if (auto val_ = object.getDouble(17)) {
+          info.fractionOnCycle = *val_;
+        }
+
+        // 18 - Heat Pump Fan Delay Time
+        if (auto val_ = object.getDouble(18)) {
+          info.hpDelayTime = *val_;
+        }
+      }
+
+      // AIRLOOPHVAC:UNITARYHEATPUMP:WATERTOAIR is not wrapped
+    }
+    return result;
+  }
+
```

Comment:
This is all a part of pre scanning for coil latent changes.

values for cycling rate, heat pump time and fan delay time are retrieved from the parent unitary.
A Curve is only created for the Coil:Heating/Cooling:WaterToAirHeatPump:EquationFit, all other coils already have one.